packages/cfd/src/Components/cfd-personal-details-form.tsx

```diff
@@ -157,6 +153,8 @@ const validatePersonalDetails = ({
     const field_error_messages = (field_name: string): string[] => [
         localize('{{field_name}} is required', { field_name }),
         localize('{{field_name}} is not properly formatted.', { field_name }),
+        // this rule is specifically for tax_identification_number when tax_residence is empty
```

Comment:
**Clarification:** Does these checks execute in an order? If so, then if tax_residence value is missing and the TIN format is incorrect, wont the TIN format error message be displayed? But in reality shouldn't the missing tax_residence be prioritized?

Reply:
They're in order accordance with the array index from `validations` object right above in this file, there's a new rule being added just to address this message